### PR TITLE
Fetch windows image from remote http server

### PR DESCRIPTION
**Pull request by cynepco3hahue**
**What this PR does / why we need it**:
Fetch windows image from remote http server to CI nodes.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@danielBelenky Can you please take a look if from std-ci perspective I configure mounts right?

**Comment by danielBelenky** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
@danielBelenky I see under windows test
```
Error response from daemon: invalid mount config for type "bind": bind source path does not exist
```
it means that source directory does not exist, do I miss something under std-ci configuration?

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
ci test please

**Comment by cynepco3hahue** [comment]
ci test please

File: automation/test.sh [hunk 1]
```diff
@@ -43,14 +43,54 @@ else
 fi
 
 export KUBEVIRT_NUM_NODES=2
-export NFS_WINDOWS_DIR=${NFS_WINDOWS_DIR:-/home/nfs/images/windows2016}
+export WINDOWS_NFS_DIR=${WINDOWS_NFS_DIR:-/var/lib/stdci/shared/images/windows2016}
+export WINDOWS_LOCK_PATH=${WINDOWS_LOCK_PATH:-/tmp/download_windows_image.lock}
```
Comment (danielBelenky) [inline automation/test.sh]: /tmp/ is not shared among the containers, why not put the lock in the shared dir? Or something exports WINDOWS_LOCK_PATH before this code runs? (if so, I couldn't find where...)

File: automation/test.sh [hunk 2]
```diff
@@ -43,14 +43,54 @@ else
...
+
+if [[ $TARGET =~ windows.* ]]; then
+  # Create images directory
+  if [[ ! -d $WINDOWS_NFS_DIR ]]; then
+    mkdir -p $WINDOWS_NFS_DIR
+  fi
+
+  # Download windows image
+  if wait_for_windows_lock; then
+    if [[ ! -f "$WINDOWS_NFS_DIR/disk.img" ]]; then
```
Comment (danielBelenky) [inline automation/test.sh]: What if we update the image? Is it a possible scenario? if so, maybe we can compare image checksum or something?

File: automation/test.sh (hunk 1 above)
Comment (cynepco3hahue) [reply automation/test.sh]: not thought about it, thx will fix it

File: automation/test.sh (hunk 2 above)
Comment (cynepco3hahue) [reply automation/test.sh]: I think we can add it in the future, if we will have a new image, currently not really relevant

File: automation/test.sh [hunk 3]
```diff
@@ -43,14 +43,54 @@ else
 fi
 
 export KUBEVIRT_NUM_NODES=2
-export NFS_WINDOWS_DIR=${NFS_WINDOWS_DIR:-/home/nfs/images/windows2016}
+export WINDOWS_NFS_DIR=${WINDOWS_NFS_DIR:-/var/lib/stdci/shared/images/windows2016}
```
[the remark this answers is not in the document]
Comment (cynepco3hahue) [reply automation/test.sh]: agree, will change the name

**Pull request merged**